README.md

```diff
@@ -12,30 +15,107 @@ is simpler and easier to work with. Additionally, for developers familar with
 the Go programming language will be instantly comfortable with Gunk files and
 syntax.
 
+### Contents
+
+* [How it works](#how-it-works)
+* [Features](#features)
+* [Installing](#installing)
+* [Usage](#usage)
+* [Contributing](#contributing)
+
+## How it works
+
+Gunk works by using Go's standard library `go/ast` package (and related
+packages) to parse all `*.gunk` files in a "Gunk Package" (and applying the
+same kinds of rules as Go package), building the appropos protobuf messages. In
+turn, those are passed to `protoc-gen-*` tools, along with any passed options.
+
+## Features
+
+- Written in simple and idiomatic [Go][go-project]
+- Intuitive and [easy to use](#usage)
+- Works on Linux, Mac and Windows
```

Comment:
Seems to me like this section doesn't add a lot of meat to the README. The first point repeats what the first few paragraphs say. The second just links to "usage". The third does add extra info, but we could just add system requirements to the installation section.